spec/services/food_service_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+RSpec.describe FoodService do
+  describe '::get_food_search', :vcr do
+    it 'gets data for food names that match a keyword(s)' do
+      response = FoodService.get_food_search('sweet potatoes')
+      
+      expect(response[:totalHits]).to be > 30000
```

Comment:
I like this assertion because it's not as flaky as testing the exact value, which could change, but it also doesn't just test the data type to have more precision in know the result was more or less what we expected!